code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -921,7 +921,8 @@
 	qdel(src)
 
 /obj/item/weapon/spellbook/oneuse/random/New()
-	var/real_type = pick(subtypesof(/obj/item/weapon/spellbook/oneuse))
+	var/static/banned_spells = list(/obj/item/weapon/spellbook/oneuse/mimery_one,/obj/item/weapon/spellbook/oneuse/mimery_two)
+	var/real_type = pick(subtypesof(/obj/item/weapon/spellbook/oneuse) - banned_spells)

```

Comment:
Instead of banning these spells, you should just give people that read them the mime powers and mimery. So they can break their new vow of silence if they want, but they'll lose their sweet powers